StudCity.Frontend/src/UI/GoogleButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function GoogleButton() {
+    return (
+		<button type='button'  className='flex break-inside bg-white text-[#486645] border-2 border-[#486645] rounded-3xl px-6 py-3 mb-4 w-full   h-16 '>
```

Comment:
I see a lot of using of color #486645, please add it to tailwind.config.js, and use a name for it(in other files as well such as Button, TextField etc).
And remove unneeded spaces before h-15 class